backstop_data/engine_scripts/puppeteer/login.js

```diff
@@ -0,0 +1,31 @@
+module.exports = async (page, scenario) => {
+  if (scenario.user === undefined) return;
+
+  var user = require('./user')(scenario);
+
+  console.log(user.email);
+  console.log(user.password);
```

Comment:
Er... where exactly is `user.password` logged (and why)?

Reply:
Copied from the `backstop_data/engine_scripts/casperjs/login.js` script. Probably shouldn't do that 🤦‍♂ I'll take it out.